cranelift/codegen/meta/src/shared/instructions.rs

```diff
@@ -3113,11 +3114,15 @@ pub(crate) fn define(
 
         The input and output types must be storable to memory and of the same
         size. A bitcast is equivalent to storing one type and loading the other
-        type from the same address.
+        type from the same address, using the specified MemFlags.
```

Comment:
s/using/both using/, just for clarity (other possibilities here include storing with one `MemFlags` and loading with another, which isn't what we're emulating but someone might imagine this  instead)